Dockerfile

```diff
@@ -4,6 +4,7 @@ RUN mkdir -p /app
 WORKDIR /app
 
 RUN apt-get update && apt-get install -y git
+RUN apt-get install -y gcc
```

Comment:
Needed to install dask

Reply:
(which includes both git and build-essentials, including gcc)